routing/transit_graph.cpp

```diff
@@ -0,0 +1,230 @@
+#include "routing/transit_graph.hpp"
+
+#include "indexer/feature_altitude.hpp"
+
+namespace
+{
+using namespace routing;
```

Comment:
Just move anonymous namespace to the routing namespace and get rid of this using namespace routing?

Reply:
done